analysis/measures_definition_pf_codes_conditions.py

```diff
@@ -173,7 +191,7 @@
         measures.define_measure(
             name=f"count_{condition_name}_by_{breakdown}",
             numerator=numerator,
-            denominator=denominator,
+            denominator=denominators[condition_name],
```

Comment:
this looks like a great way to do it! but I think we also need to add this to the denominaotr of the overall measure?

```py
    measures.define_measure(
        name=f"count_{condition_name}",
        numerator=numerator,
        denominator=pf_condition_denominators[condition_name],
        intervals=months(monthly_intervals).starting_on(start_date),
    )
```

also updating the suggested name change here:

```suggestion
            denominator=pf_condition_denominators[condition_name],
```